experimental/SetPartitions/test/LinearComb-test.jl

```diff
@@ -0,0 +1,27 @@
+@testset "LinearCombinations of SetPartitions" begin
```

Comment:
This testfile is named `LinearComb-test.jl`, although it tests the features from `LinearPartition.jl`. That seems to be a little mismatch.